js/manifest.js

```diff
@@ -11,8 +11,12 @@ fin.desktop.Application.getCurrent().getManifest(function (manifest) {
       window.localStorage.setItem('notificationsVersion', "V2");
       window.localStorage.setItem('notificationsHeight', 60);
     }
+	
+    let popoutChatNotificationFlag = manifest.startup_app.customData.popoutChatOnNotificationClick;
```

Comment:
can you add a check here for if `customData` does not exist

Reply:
It's in an if block that already does that